src/motor_hardware.cc

```diff
@@ -173,6 +177,24 @@ void MotorHardware::setWheelJointVelocities(double leftWheelVelocity, double rig
     return;
 }
 
+// Publish motor state conditions
+void MotorHardware::publishMotorState(void) {
+    ubiquity_motor::MotorState mstateMsg;
+
+    mstateMsg.header.frame_id = "motor_state";
```

Comment:
If you really want it to have a frame make it "base_link" but otherwise the data in the message is frameless so we should have an empty string.

Reply:
Ok.  Silly me.   This is the same sort of frame_id used for fiducials in that it is a spacial frame.  I'll make it empty.
I was mostly focused on timestamp and just put this in there not thinking much about what it would be used for later.